catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java

```diff
@@ -150,44 +154,53 @@ private void copyFFmpegBinary(final InputStream inputStream) throws IOException
     public CreateStorageResponse process(final CreateStorageResponse input)
             throws PluginExecutionException {
         // TODO: How to handle application/octet-stream?
-        try {
-            processContentItems(input.getCreatedContentItems(), input.getProperties());
-        } catch (IllegalArgumentException e) {
-            input.getProcessingErrors()
-                    .add(new ProcessingDetailsImpl(this.getClass()
-                            .getName(), e));
-        }
+        processContentItems(input.getCreatedContentItems(), input.getProperties());
         return input;
     }
 
     @Override
     public UpdateStorageResponse process(final UpdateStorageResponse input)
             throws PluginExecutionException {
         // TODO: How to handle application/octet-stream?
-        try {
-            processContentItems(input.getUpdatedContentItems(), input.getProperties());
-        } catch (IllegalArgumentException e) {
-            input.getProcessingErrors()
-                    .add(new ProcessingDetailsImpl(this.getClass()
-                            .getName(), e));
-        }
+        processContentItems(input.getUpdatedContentItems(), input.getProperties());
         return input;
     }
 
     private void processContentItems(final List<ContentItem> contentItems,
-            final Map<String, Serializable> properties)
-            throws PluginExecutionException, IllegalArgumentException {
+            final Map<String, Serializable> properties) throws PluginExecutionException {
         Map<String, Map<String, Path>> tmpContentPaths =
                 (Map<String, Map<String, Path>>) properties.get(Constants.CONTENT_PATHS);
 
         for (ContentItem contentItem : contentItems) {
             if (isVideo(contentItem)) {
-                Map<String, Path> contentPaths = tmpContentPaths.get(contentItem.getId());
 
+                final long contentItemSizeBytes;
+                try {
+                    contentItemSizeBytes = contentItem.getSize();
+                } catch (IOException e) {
+                    LOGGER.warn(
+                            "Error retrieving size for ContentItem (id={}). Skipping creating thumbnail for metacard (id={})");
+                    break;
+                }
+
+                final long bytesPerMegabyte = 1024L * 1024L;
+                final long maxFileSizeBytes = maxFileSizeMB * bytesPerMegabyte;
```

Comment:
I decided against doing that so I can keep
```java
static final int DEFAULT_MAX_FILE_SIZE_MB = 120;
```
without having to do some calculation to set the maxFileSizeBytes default, like
```java
private int maxFileSizeBytes = DEFAULT_MAX_FILE_SIZE_MB * BYTES_PER_MEGABYTE;
```